src/screens/Home/OnboardingScreen/StepScreens.tsx

```diff
@@ -11,54 +11,55 @@ import {
 } from 'react-native'
 import styles from './stepStyles'
 
-/* AKA: Q&A screen */
-export default function OnboardingScreen(props: Props) {
+export default function StepScreens(props: Props) {
   const {navigate} = props.navigation
   const [step, setStep] = useState(2)
   const [text, setText] = useState('Select questions to answer')
-  const [header, setHeader] = useState('Step 1')
-  const [textStyle, setTextStyle] = useState(styles.stepOne)
+  const [title, setTitle] = useState('Step 1')
+  const [textStyle, setTextStyle] = useState(styles.textOne)
+  const [imageSource, setImageSource] = useState(require('../../../../assets/images/step1.png'))
+  const [buttonContent, setButtonContent] = useState('Next')
 
   return (
     <View style={styles.container}>
-      <View style={styles.main}>
-        <Text style={styles.header}>{header}</Text>
+      <View style={styles.header}>
+        <Text style={styles.title}>{title}</Text>
         <Text style={textStyle}>{text}</Text>
```

Comment:
Create a generic text style since your styles textOne, textTwo, textThree, textFour have a ton of similarity. Save the styles unique to each step to their respective styles (refer to below comment for more)
```js
<Text style={[styles.text, textStyle]}>{text}</Text>

```